cmd/algorand-indexer/daemon.go

```diff
@@ -133,6 +136,19 @@ var daemonCmd = &cobra.Command{
 				fmt.Fprint(os.Stderr, "missing indexer data directory")
 				os.Exit(1)
 			}
+
+			// sync local ledger
```

Comment:
Not required now, but I think we'll need to figure out a way to put this in an Init function thats part of the block processor.

There is also an edge case we could check to fail fast: if the catchpoint is ahead of nextDBRound we should avoid fast catchup, and probably inform the user that they might want to finish catchup with an earlier version of Indexer.